src/app/shared/formly/wrappers/horizontal-wrapper/horizontal-wrapper.component.html

```diff
@@ -1,10 +1,11 @@
 <div class="form-group row" [class.has-error]="showError" [attr.data-testing-id]="keyString + '-wrapper'">
   <label
+    *ngIf="props.label"
     [attr.for]="id"
     class="col-form-label"
     [ngClass]="props.labelClass ? props.labelClass : dprops.labelClass"
-    *ngIf="props.label"
-    >{{ props.label | translate }}<span class="required" *ngIf="props.required && !to.hideRequiredMarker">*</span>
+    >{{ props.labelNoTranslate ? props.label : (props.label | translate)
```

Comment:
the labelNoTranslate property should be available in the horizontal-checkbox-wrapper as well and be documented in the formly.md documentation

Reply:
Done